cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -1442,7 +1445,7 @@ def test_create_course_with_course_creator(self):
             auth.add_users(self.user, CourseCreatorRole(), self.user)
             self.assert_created_course()
 
-    def test_create_course_with_unicode_in_id_disabled(self):
+    def test_create_course_with_text_type_in_id_disabled(self):
```

Comment:
Given that the `ALLOW_UNICODE_COURSE_ID` setting name didn't change, probably should leave the test name as it was.

Reply:
Bah, bad replace. Will fix.